fs/partition/fs_gpt.c

```diff
@@ -393,6 +393,7 @@ int parse_gpt_partition(FAR struct partition_state_s *state,
   int nb_part;
   int count;
   int ret;
+  blkcnt_t lastlba = gpt_last_lba(state);
```

Comment:
move before line 393?

Reply:
Sure. I also initialize it close to where it is used, looks cleaner that way imho.